WPFSKillTree/ViewModels/BuildsControlViewModel.cs

```diff
@@ -196,7 +196,7 @@ public ISkillTree SkillTree
         public IReadOnlyList<ClassFilterItem> ClassFilterItems { get; }
 
         public BuildsControlViewModel(IExtendedDialogCoordinator dialogCoordinator, IPersistentData persistentData,
-            ISkillTree skillTree)
+            ISkillTree skillTree, IAscendancyClasses ascendancyClasses)
```

Comment:
Right. It looks like I've left some inconsistency while playing with different interface dependencies.